tests/cycpp_tests.py

```diff
@@ -585,16 +585,19 @@ def test_schemafilter():
         '    "<interleave>\\n"\n'
         '    "    <element name=\\"x\\">\\n"\n'
         '    "        <oneOrMore>\\n"\n'
-        '    "            <element name=\\"key\\">\\n"\n'
-        '    "                <data type=\\"int\\"/>\\n"\n'
-        '    "            </element>\\n"\n'
-        '    "            <element name=\\"val\\">\\n"\n'
-        '    "                <data type=\\"double\\"/>\\n"\n'
+        '    "            <element name=\\"item\\">\\n"\n'
+        '    "                <interleave>\\n"\n'
```

Comment:
Moreover, doesn't that make it somewhat harder to process since you'll be relying on other information to determine which is the key and which is the value, rather than relying on order?